cicd/forgeops-tests/tests/smoke/idm/test_idm.py

```diff
@@ -8,28 +8,28 @@
 Test are sorted automatically so that's why it's needed to keep test_0[1,2,3]_ naming.
 """
 # Lib imports
-import json
-
-import unittest
 from requests import get, post, put, delete, session
 
 # Framework imports
-from config.ProductConfig import IDMConfig
+from ProductConfig import IDMConfig
+from utils import logger, rest
 
 
-class IDMSmoke(unittest.TestCase):
+class TestIDM(object):
     idmcfg = IDMConfig()
     testuser = '/forgeops-testuser'
 
-
     def test_0_ping(self):
         """Pings OpenIDM to see if server is alive using admin headers"""
-        resp = get(verify=self.idmcfg.ssl_verify, auth=('openidm-admin', 'openidm-admin'),
-                   url=self.idmcfg.rest_ping_url)
-        self.assertEqual(resp.status_code, 200)
+
+        logger.test_step('Ping OpenIDM')
+        response = get(verify=self.idmcfg.ssl_verify, auth=('openidm-admin', 'openidm-admin'),
+                       url=self.idmcfg.rest_ping_url)
+        rest.check_http_status(http_result=response, expected_status=200)
 
     def test_1_create_managed_user(self):
         """Test to create managed user as admin"""
+
         headers = self.idmcfg.get_admin_headers({
```

Comment:
Future work: Would be good to use this get_ccccc_headers consistently for the other products too.

Reply:
I don't think other products pass headers the same way that IDM does.

Reply:
This appears in test_am.py
`    def setup_class(cls):
        """Setup a user that will be tested in user login."""

        headers = {'X-OpenAM-Username': 'amadmin', 'X-OpenAM-Password': 'password',
                   'Content-Type': 'application/json', 'Accept-API-Version': 'resource=2.0, protocol=1.0'}`

It will be good to try and be consistent.